multinet/api/views/session.py

```diff
@@ -0,0 +1,39 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import viewsets
+from rest_framework.response import Response
+
+from ..models import Network, Session, Workspace
+from .serializers import SessionSerializer
+
+
+class SessionViewSet(viewsets.ModelViewSet):
+    queryset = Session.objects.all()
```

Comment:
Does this pull all rows from the DB when the class is created?

Reply:
No, `QuerySet` objects are a sort of "thunk" that build up a representation of a DB query without executing it (until needed). So here I'm setting up a base query that retrieves all session objects, but it gets refined down in, for example, the `list()` function before the database query is executed.

Or rather, it's supposed to--looks like I forgot to include the logic for that. It would be something like `self.get_queryset().filter(network=network)` or similar.